app/lib/screens/signing/sign_with_qrcode.dart

```diff
@@ -0,0 +1,327 @@
+              decoration: InputDecoration(
+                hintText: 'Scan a QR code to see the data here...',
+                errorText: scannedDataError,
+                border: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(8),
+                ),
+                suffixIcon: _textController.text.isNotEmpty
+                    ? IconButton(
+                        icon: const Icon(Icons.copy),
+                        onPressed: () {
+                          Clipboard.setData(
+                              ClipboardData(text: _textController.text));
+                          ScaffoldMessenger.of(context).clearSnackBars();
+                          ScaffoldMessenger.of(context).showSnackBar(
+                              const SnackBar(content: Text('Copied!')));
+                        },
+                      )
+                    : null,
+              ),
+            ),
+            const SizedBox(height: 24),
+            Text(
+              'Select Wallet',
+              style: Theme.of(context).textTheme.titleLarge?.copyWith(
+                    fontWeight: FontWeight.bold,
+                    color: Theme.of(context).colorScheme.onSurface,
+                  ),
+            ),
+            const SizedBox(height: 16),
+            DropdownButtonFormField<Wallet>(
+              value: selectedWallet,
+              decoration: InputDecoration(
+                border: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(8),
+                ),
+                hintText: 'Select a wallet',
+                errorText: walletError,
+              ),
+              items: wallets.map((wallet) {
+                return DropdownMenuItem(
+                  value: wallet,
+                  child: Text(wallet.name,
+                      style: Theme.of(context).textTheme.bodyMedium?.copyWith(
+                            color: Theme.of(context).colorScheme.onSurface,
+                          )),
+                );
+              }).toList(),
+              onChanged: (Wallet? value) {
+                setState(() {
+                  selectedWallet = value;
+                  if (walletError != null) {
+                    _validateInputs();
+                  }
+                });
+              },
+            ),
+            const SizedBox(height: 32),
+            ElevatedButton(
+              onPressed: isLoading ? null : _signText,
+              style: ElevatedButton.styleFrom(
+                padding: const EdgeInsets.all(16),
+                shape: RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(8),
+                ),
+              ),
+              child: isLoading
+                  ? const SizedBox(
+                      height: 20,
+                      width: 20,
+                      child: CircularProgressIndicator(strokeWidth: 2),
+                    )
+                  : const Text('Sign Text'),
```

Comment:
please add style for this text
```suggestion
                  : const Text('Sign'),
```